code-challenges.js

```diff
@@ -12,26 +12,65 @@ var num2 = 0
 var num3 = -7
 // Expected output: "-7 is not divisble by three"
 
+const divide = (num) => {
+  if (num % 3 === 0) {
+    return `"${num} is divisible by three"`
+// I had single qoutes and not back ticks..... ***
+  }else{
+    return `"${num} is not divisible by three"`
+  }
+}
+console.log(divide(num1));
+console.log(divide(num2));
+console.log(divide(num3));
 
 
 
-
-// --------------------2) Create a function that takes in the randomNouns variable and returns an array with all the words capitalized.
+// // --------------------2) Create a function that takes in the randomNouns variable and returns an array with all the words capitalized.
 
 var randomNouns = ["streetlamp", "potato", "teeth", "conclusion", "nephew", "temperature", "database"]
 // Expected output: ["Streetlamp", "Potato", "Teeth", "Conclusion", "Nephew", "Temperature", "Database"]
 
-
-
+// need a const that returns every variable capitalized
+const capital = (array) => {
+  let array42 = array.map(value => {
+    return value[0].toUpperCase() + value.substring(1)
+    //in order for me to signal to the function JUST TO capitalize the first letter, I use index[0] and I want the entire word after capitalization as well to complete my answer as .substring[1] to signal that I need all the letters
+  })
+  return array42.join(" ")
```

Comment:
Be sure to read, reread, complete the challenge and then DOUBLE CHECK your challenge requirements. 

> Create a function that takes in the randomNouns variable and **returns an array** with all the words capitalized.